doc/source/user/usage.rst

```diff
@@ -77,9 +77,12 @@ input file might look as follows:
         Tsolid_sf = 1e-3
     []
 
-If the input file is ``sam_template.inp``, the SAM code will rely the general MOOSE plugin that can be created as::
+If the templated input file is ``sam_template.inp``, the SAM code will rely the general MOOSE plugin that can be created as::
 
-    moose_plugin = watts.PluginMOOSE('sam_template.inp')
+    moose_plugin = watts.PluginMOOSE('sam_template.inp', supp_inputs = [])
```

Comment:
The default for `supp_inputs` is an empty list, so it doesn't appear that there's any purpose to explicitly specifying that here. I would just list a different example after the next blurb that shows `supp_inputs` being used to specify an additional file.